moveit_core/trajectory_processing/src/ruckig_traj_smoothing.cpp

```diff
@@ -76,9 +76,9 @@ bool RuckigSmoothing::applySmoothing(robot_trajectory::RobotTrajectory& trajecto
 
   // Instantiate the smoother
   double timestep = trajectory.getAverageSegmentDuration();
-  ruckig::Ruckig<0> ruckig{ num_dof, timestep };
-  ruckig::InputParameter<0> ruckig_input{ num_dof };
-  ruckig::OutputParameter<0> ruckig_output{ num_dof };
+  ruckig::Ruckig<0> ruckig(num_dof, timestep);
+  ruckig::InputParameter<0> ruckig_input(num_dof);
+  ruckig::OutputParameter<0> ruckig_output(num_dof);
```

Comment:
Sweet, I hope this works.  I had no idea there would be a way to fix this without waiting for the next release.

Reply:
nope, it doesn't. Did the last Ruckig release fail for Galactic?

Reply:
I don't think so because it is in galactic-testing, I think the last release just missed the sync.

Reply:
How did https://github.com/ros-planning/moveit2/pull/571 pass then? Also, there were a couple of main builds that succeeded with Galactic

Reply:
[Ruckig was just synced](https://discourse.ros.org/t/new-packages-for-galactic-geochelone-2021-09-20/22374), so this should be fixed. I'm removing the changes from this PR.